azurelinuxagent/ga/exthandlers.py

```diff
@@ -1458,6 +1459,22 @@ def set_extension_resource_limits(self):
             extension_name=extension_name, cpu_quota=resource_limits.get_extension_slice_cpu_quota())
         CGroupConfigurator.get_instance().set_extension_services_cpu_memory_quota(resource_limits.get_service_list())
 
+    def log_telemetry_if_ext_uses_resource_limits(self):
+        extension_name = self.get_full_name()
+        man = self.load_manifest()
+        resource_limits = man.get_resource_limits()
+
+        def _log():
+            event.info(WALAEventOperation.ExtensionResourceGovernance,'{0} is using resource governance', extension_name)
+
+        if resource_limits is None:
+            return
+
+        if resource_limits.get_extension_slice_cpu_quota() is not None or resource_limits.get_extension_slice_memory_quota() is not None:
```

Comment:
Also log whether the extension is setting limits and/or specifying a service list? (to differentiate between the 2 cases)